sway/input/cursor.c

```diff
@@ -1025,7 +1029,8 @@ static void dispatch_cursor_axis(struct sway_cursor *cursor,
 	}
 
 	wlr_seat_pointer_notify_axis(cursor->seat->wlr_seat, event->time_msec,
-		event->orientation, event->delta, event->delta_discrete, event->source);
+		event->orientation, scroll_factor * event->delta,
+		scroll_factor * event->delta_discrete, event->source);
```

Comment:
`event->delta_discrete` is an integer, so maybe it should be rounded instead of being silently floored.